tests/golden/upgrade-notification/openshift4-console/openshift4-console/31_upgrade_notification.yaml

```diff
@@ -125,8 +126,11 @@ data:
     set -euo pipefail
 
     echo "OVERLAY_VERSION_MINOR: $OVERLAY_VERSION_MINOR"
-    CURRENT_MINOR=$(oc version -oyaml | yq '.openshiftVersion' | cut -d'.' -f2)
-    echo "CURRENT_MINOR: $CURRENT_MINOR"
+    # NOTE(sg): We use 4.0.0 as fallback current version since we only care that
```

Comment:
If I understand correctly how you'd like this to work, this will require that we remove the postsync hook when there's no future channel overlay present (which currently isn't the case).